### PR TITLE
Revives Syndiestation

### PR DESCRIPTION
**Pull request by tralezab**
It's still syndiestation, but i'm now the lone worker on it

### CODE REVIEW

**Comment by tralezab** [comment]
Manuel Ready

File: config/game_options.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## HEALTH ###
+f## HEALTH ###
```
Comment (Paxilmaniac) [inline config/game_options.txt:1]: Was the "f" in front of ## HEALTH ## intentional?

Comment (tralezab) [reply config/game_options.txt:1]: you did this, not me!!!

**Comment by tralezab** [comment]
new stuff maybe?

**Comment by tralezab** [comment]
aha i knew it

**Comment by tralezab** [comment]
wait shit

**Comment by tralezab** [comment]
missed compile error

**Comment by tralezab** [comment]
last time i should have to do this ;)

**Comment by Jared-Fogle** [comment]
arent you supposed to close this right after so we dont have it on the tracker

**Pull request closed without merging**